OtripleS.Web.Api/Models/Students/Student.cs

```diff
@@ -4,6 +4,22 @@ namespace OtripleS.Web.Api.Models.Students
 {
     public class Student : IAuditable
     {
+        public Student()
+        {
+        }
+
+        public Student(Guid id, string firstName, string middleName, string lastName, DateTimeOffset birthDate, Gender gender)
+        {
+            Id = id;
+            FirstName = firstName;
+            MiddleName = middleName;
+            LastName = lastName;
+            BirthDate = birthDate;
+            Gender = gender;
+
+            CreatedDate = DateTime.UtcNow;
```

Comment:
I don't think we need these constructors either.

Reply:
granted, in an anemic domain model you don't (which this seems to be heading towards). The parameterless ctor is required for the Filler library, otherwise it wouldn't be there. If an anemic domain model is indeed the direction of this model, I'll remove the ctor